### PR TITLE
Resolves #2467 and #2046 : Strings marked for I18N in templates

### LINKED ISSUE #2467: Mark strings in views/templates that would be locale-specific
Parent Issue: #2046

All strings in views/templates which will need translation needs to be marked.

### PR DESCRIPTION
**Pull request by sainiak009**
I have scanned through all the templates and tried my best to mark all strings. Please review and let me know if any error of missing strings. New .po/.pot files are generated. I have generated translations from Google Translator Toolkit, Most of the strings are converted but few are left and can be edited manually by downloading .po file by a volunteer.

### CODE REVIEW

**Comment by sainiak009** [comment]
@niranjan94 Please have a look.

File: app/templates/404.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>You got 404'd - HTTP 404 - Page Not Found - Open Event</title>
+    <title>{{_('You got')}} 404'd - HTTP 404 - {{_('Page Not Found')}} - {{_('Open Event')}}</title>
```
Comment (niranjan94) [inline app/templates/404.html]: Don't mark **Open Event** for translation. That would be name of the service. EventYay.

Comment (mariobehling) [inline app/templates/404.html]: Yes, the name of the service is generic. Someone else could set it up using a different name for example.

Comment (sainiak009) [inline app/templates/404.html]: @mariobehling @niranjan94 "Open event" is removed from marked strings

**Comment by sainiak009** [comment]
@niranjan94 @SaptakS Please review and merge

**Pull request merged**